rtgui/locallabtools2.cc

```diff
@@ -8491,16 +8540,15 @@ Locallabcie::Locallabcie():
     Gtk::Label* modeLabelbwev = Gtk::manage(new Gtk::Label(M("TP_LOCALLAB_SIGMOIDQJ") + ":"));
     modeHBoxbwev->pack_start(*modeLabelbwev, Gtk::PACK_SHRINK);
 
-    bwevMethod->append(M("TP_LOCALLAB_BWEVNONE"));
     bwevMethod->append(M("TP_LOCALLAB_BWEVSIG"));
+    bwevMethod->append(M("TP_LOCALLAB_BWEVSLOP"));
     bwevMethod->set_active(1);
     bwevMethodConn = bwevMethod->signal_changed().connect(sigc::mem_fun(*this, &Locallabcie::bwevMethodChanged));
     modeHBoxbwev->pack_start(*bwevMethod);
 
     comprBox->pack_start(*comprcie);
     comprBox->pack_start(*strcielog);
     comprBox->pack_start(*satcie);
-    comprBox->pack_start(*logcieq);
```

Comment:
Remove all references to `logcieq`.

Reply:
OK but see my post on free software and compatibility